src/internal/io.js

```diff
@@ -77,6 +77,9 @@ function getFnCallDesc(meth, fn, args) {
   } else if(fn.fn) {
     ({context, fn} = fn)
   }
+  if (context && is.string(fn)) {
+    fn = context[fn]
+  }
   check(fn, is.func, `${meth}: argument ${fn} is not a function`)
```

Comment:
In case a wrong string is provided, since `fn` was mutated in the previous line, the message will print smth like `argument undefined is not a function`. perhaps better to keep the string name for more helpful messages (since it's more likely that an error could happen with string args)